src/components/InputV2/PrivateKey.tsx

```diff
@@ -15,9 +15,11 @@ export const PrivateKeyInput = ({
   ...baseProps
 }: PrivateKeyInputProps) => {
   const [hasFocus, setHasFocus] = useState<boolean>(false)
+  const [hasChanged, setHasChanged] = useState<boolean>(false)
```

Comment:
Omitting this means the input would have a different behaviour around error handling than the other ones. We added this so that we only show errors after the user actually used the input, otherwise it would seem like inputs had errors way too fast.

Same for `SeedPhraseInput`